unicorn-fail/dreditor
# Add support for templates (macros)

This is a first stab to restart [Issue Macros and Templates](https://drupal.org/node/1115636)

As this is a WIP please give it a try.
## Testing
1. Navigate to any Drupal core issue : https://drupal.org/node/2160555
2. You should see ![node-view-comment-form](https://cloud.githubusercontent.com/assets/371014/3208572/62117f02-ee3c-11e3-8379-be7169a8c623.png)
3. Click on `Macro & Templates`
   1. Click on `Drupal Core`
   2. Select Macro & Templates > Drupal code > Gates > Documentation
   3. Select 'Gates' ![drupal-gates](https://cloud.githubusercontent.com/assets/371014/3208598/cc2c6aaa-ee3c-11e3-9131-f702c0ec4e29.png)
   4. Hover over `Add issue summary` and note it is about to set `Status` and `Tags` 
   5. Click on `Add issue summary`
   6. Check for status change and tags
4. Click on `Macro & Templates`
   1. Click on `_default`
   2. Click on last item 'view' or 'edit' to check or edit the stock responses.
## Open issues
- [x] Fix 'view' or 'edit'
- [x] It adds itself through commit.message.js which already has a generalize #todo
- [x] Remove old links
- [x] Add to node/*/edit page
- [x] As node/*/edit has the issue _fields_ state changingmacro's are broken.
